k8s/staging/runners/protected/windows/x86_64/v2/release.yaml

```diff
@@ -57,27 +57,22 @@ spec:
           pre_build_script = """
           Write-Output 'Executing Spack pre-build setup script'
 
-          $PY3 = $null
-          $pythonCmds = @('python3', 'python')
-
-          foreach ($cmd in $pythonCmds) {
-              if (Get-Command -ErrorAction SilentlyContinue $cmd) {
-                  $PY3 = Get-Command $cmd
-                  break
-              }
-          }
-
-          if (-not $PY3) {
-              Write-Output "Unable to find python3 executable"
-              exit 1
+          $a=(get-command -ErrorAction SilentlyContinue fake)
```

Comment:
Why remove the exit and silently continue?

Reply:
The silently continue is just so `Get-Command` doesn't fail, it's scoped to that commandlet.
Totally missed that we were failing early on no Python, will add.

Reply:
Added